GitCommands/Git/GitGpgController.cs

```diff
@@ -232,13 +232,13 @@ public string GetCommitVerificationMessage(GitRevision revision)
                 return null;
             }
 
-            var module = GetModule();
+            // '--raw' returns info in stderr
             GitArgumentBuilder args = new("verify-tag")
             {
                 { raw, "--raw" },
                 tagName
             };
-            return module.GitExecutable.GetOutput(args);
+            return GetModule().GitExecutable.Execute(args).StandardError;
```

Comment:
https://github.com/gitextensions/gitextensions/issues/10242#issuecomment-1279707819

Reply:
stderr has info without error exit is set.
what is the suggestion?